### ISSUE TITLE
tsc.exe scans directory even if tsconfig specifies files

### ISSUE THREAD

**Issue by arogg** [Microsoft/TypeScript#8349]
Steps I did:

- create VS2015 project (i did MVC4 empty project)
- create file1.ts with any valid TS content (like var o = 3;) and add this to project
- create tsconfig.json with content (and add to project):
```
{
  "compilerOptions": {
    "noImplicitAny": true,
    "removeComments": false,
    "module": "amd",
    "target": "es5",
    "outFile": "compiled.js"
  },
  "files": [ "file1.ts" ]
}
```
-  hard to give an explanation here how to reproduce this step, since manually creating empty folders did not work: I just copied a folder with a very deep structure (with nodemodules and bowermodules in it) to the project directory. do NOT add this folder to the project however. 
 - build project (i'm assuming tsc will run now, it does for me)
- tsc will error complaing about path too long (although the folder was NOT add to the project):C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\TypeScript\Microsoft.TypeScript.targets(173,5): error MSB6003: The specified task executable "tsc.exe" could not be run. The specified path, file name, or both are too long. The fully qualified file name must be less than 260 characters, and the directory name must be less than 248 characters.
- bottom line is: tsc should not in ANY way be scanning the file system if the tsconfig is specifying the files explicitely. i am ASSUMING it is scanning the file system, otherwise it could not have found out that paths are too long.

- i have ts 1.8.6

**Comment by mhegazy** [comment]
do you have an import statement in `file1.ts`?

**Comment by arogg** [comment]
no only code in the file is
`var o = 3;`

**Comment by mhegazy** [comment]
when you run tsc.exe manually. dose it show the same error?

**Comment by mhegazy** [comment]
closing for now. please reopen if you have more information you can share. ideally a sample project that reproduces the issue.
